app/models/content_item_factory.rb

```diff
@@ -1,12 +1,16 @@
 class ContentItemFactory
   def self.build(content_store_response)
-    content_item_class(content_store_response["document_type"]).new(content_store_response)
+    content_item_class(
+      content_store_response["document_type"],
+      content_store_response["schema_name"],
+    ).new(content_store_response)
   end
 
-  def self.content_item_class(document_type)
-    klass = document_type.camelize.constantize
-    klass.superclass == ContentItem ? klass : ContentItem
-  rescue StandardError
-    ContentItem
+  def self.content_item_class(document_type, schema_name)
+    if [document_type, schema_name] in %w[landing_page landing_page]
+      LandingPage
+    else
+      ContentItem
```

Comment:
I think it's fine to add a separate clause for Landing Page whilst extra content is being merged in from YAML files, however the rest of the models should be instantiated as normal as this code supports the app consolidation work that is on-going.

i.e.

```ruby
def self.content_item_class(document_type)
  if [document_type, schema_name] in %w[landing_page landing_page]
    LandingPage
  else
    klass = document_type.camelize.constantize
    klass.superclass == ContentItem ? klass : ContentItem
rescue StandardError
  ContentItem
end
```